README.md

```diff
@@ -68,6 +68,17 @@ Last Updated On: 18-04-2022
 
 ## Installation
 
+### [lazy.nvim](https://github.com/folke/lazy.nvim)
+
+```lua
+return {
+  'nvim-lualine/lualine.nvim',
+  event = { "BufRead", "BufNewFile", "VimEnter" },
+  dependencies = { 'nvim-tree/nvim-web-devicons', event = "VeryLazy" }
+  opts = {} -- lazy equivalent to setup({}) function
```

Comment:
I don't think this comment is appropriate here.
`-- lazy equivalent to setup({}) function`
This is an instruction for a specific plugin manager, not a "lazy vs packer" comparison.

If you look at the current instructions for the packer and do it in the same way, then it should be only 2 lines:
```lua
  {
    "nvim-lualine/lualine.nvim",
    dependencies = { "nvim-tree/nvim-web-devicons" },
  },
```